static/templates/settings/muted_users_settings.hbs

```diff
@@ -0,0 +1,13 @@
+<div id="muted-user-settings" class="settings-section" data-name="muted-users">
+    <input id="muted_users_search" class="search" type="text" placeholder="{{t 'Search muted users...' }}" aria-label="{{t 'Search muted users...' }}"/>
```

Comment:
We apparently have several of this bug, but we should use …, aka the unicode `...` character, not 3 dots, since it's visibly prettier.  This is not new with your PR -- but maybe another opportunity to do a quick prep commit (or just separate mini PR) to clean up.
```
p$ git grep '[.][.][.]' static/templates/
static/templates/admin_user_group_list.hbs:        <div class="input" contenteditable="true" data-placeholder="{{t 'Add member...' }}"></div>
static/templates/message_body.hbs:<div class="message_expander message_length_controller" title="{{t 'Expand message (-)' }}">{{t "[More...]" }}</div>
static/templates/settings/attachments_settings.hbs:    <input id="upload_file_search" class="search" type="text" placeholder="{{t 'Search uploads...' }}" aria-label="{{t 'Search uploads...' }}"/>
static/templates/settings/muted_topics_settings.hbs:    <input id="muted_topics_search" class="search" type="text" placeholder="{{t 'Search muted topics...' }}" aria-label="{{t 'Search muted topics...' }}"/>
static/templates/settings/muted_users_settings.hbs:    <input id="muted_users_search" class="search" type="text" placeholder="{{t 'Search muted users...' }}" aria-label="{{t 'Search muted users...' }}"/>
static/templates/typing_notification.hbs:<li data-email="{{this.email}}" class="typing_notification">{{#tr this}}__full_name__ is typing...{{/tr}}</li>
```

Reply:
I'm going to change this text to "Filter muted users" in the spirit of b40e50f2959998d0193b65a3e5b4065ff5abfea5, but I've opened #18201 for the remaining ones.

Reply:
Sounds great!